rust_src/src/buffers.rs

```diff
@@ -1843,4 +1847,80 @@ pub fn byte_char_debug_check(b: LispBufferRef, charpos: isize, bytepos: isize) {
     }
 }
 
+// True if B can be used as 'other-than-BUFFER' buffer.
+fn candidate_buffer(b: LispObject, buffer: LispObject) -> bool {
+    if let Some(buf) = Option::<LispBufferRef>::from(b) {
```

Comment:
It looks like the preferred style is to use X.into(), but I can't see a way to make that work with the Rust type system here. Any suggestions?

Reply:
`From` is ok when required.

Reply:
Thanks.